src/main/js/html.js

```diff
@@ -553,7 +566,7 @@ var backgroundColorAdjustSuffix = "BackgroundColorAdjust";
 
                 var par_edges = rect2edges(proc_e.getBoundingClientRect(), context);
 
-                applyFillLineGap(linelist, par_edges.before, par_edges.after, context);
+                applyFillLineGap(linelist, par_edges.before, par_edges.after, context,proc_e);
```

Comment:
idiomatic formatting:

```suggestion
                applyFillLineGap(linelist, par_edges.before, par_edges.after, context, proc_e);
```